src/us/kbase/workspace/database/mongo/MongoWorkspaceDB.java

```diff
@@ -191,8 +191,18 @@
 		wsVer.add(idxSpec(Fields.VER_WS_ID, 1, Fields.VER_ID, 1, Fields.VER_VER, 1, IDX_UNIQ));
 		//find versions and sort descending on version
 		wsVer.add(idxSpec(Fields.VER_WS_ID, 1, Fields.VER_ID, 1, Fields.VER_VER, -1, IDX_UNIQ));
-		//find versions by data object
-		wsVer.add(idxSpec(Fields.VER_TYPE, 1, Fields.VER_CHKSUM, 1));
+		//find versions per type and sort descending on version
```

Comment:
All of these comments use the verb "find", isn't this code adding indexes to the MongoDB collections, not executing a search?